gitpod-io/gitpod
# fix workspace file extension

#### What it does

Rename a workspace file to use the proper extension.

#### How to test

Open this branch in gitpod.io and check explorer. It should be named without the file extension and contain all folders:
<img width="360" alt="Screenshot 2021-04-06 at 14 21 23" src="https://user-images.githubusercontent.com/3082655/113709957-69b2cc00-96e3-11eb-8172-d95d7a6888cb.png">

Also if you open the workspace file you should get a proper support for the workspace file type.
